components/operator/helm/crds/templates/crds/apiextensions.k8s.io_v1_customresourcedefinition_gateways.formance.com.yaml

```diff
@@ -84,6 +84,9 @@ spec:
                       Indicates the hostname on which the stack will be served.
                       Example : `formance.example.com`
                     type: string
+                  ingressClassName:
+                    description: Custom Ingress class to use
```

Comment:
I don't think the word custom is needed. Every `Ingress` in k8s need an `IngressClass` to point to. Sometimes the cluster has a default one sometimes it doesn't.